functions/Get-DbaDbFile.ps1

```diff
@@ -50,6 +53,12 @@ function Get-DbaDbFile {
 
         Will return an object containing all file groups and their contained files for the Impromptu and Trading databases on the sql2016 SQL Server instance
 
+    .EXAMPLE
+        PS C:\> $db = Get-DbaDatabase -SqlInstance sql2016 -Database Impromptu, Trading;
```

Comment:
Semicolons are not encouraged in PowerShell as they aren't required and aesthetically pleasing.